sonar-plugin-api/src/main/java/org/sonar/api/ce/posttask/PostProjectAnalysisTask.java

```diff
@@ -80,20 +80,17 @@
      * <p>
      * This date is the same as the date of the project analysis report and the snapshot.
      *
-     * @deprecated use {@link #getAnalysisDate()} instead. When {@link #getAnalysisDate()} returns
+     * @deprecated use {@link #getAnalysis().getDate()} instead. When {@link #getAnalysis()} returns
      *             {@link Optional#empty() empty}, the current date will be returned.
      */
     @Deprecated
     Date getDate();
 
     /**
-     * Date of the analysis.
-     * <p>
-     * This date is the same as the date of the project analysis report and therefore as the analysis in DB. It can be
-     * missing when the status of the task is {@link org.sonar.api.ce.posttask.CeTask.Status#FAILED FAILED}.
-     * </p>
+     * Analysis containing the UUID of the analysis and the date
+     *
      */
-    Optional<Date> getAnalysisDate();
```

Comment:
:warning:  according to SONAR-9891, `getAnalysisDate()` must be deprecated, not removed